Governance/concepts/availability-statement.md

```diff
@@ -0,0 +1,28 @@
+# Data availability statement
+
+A Data Availability Statement is a brief document which details where supporting data are available, and how the data can be accessed and reused. It should list specific restrictions, if any. If the data underlying the study cannot be made available or reused, the Data Availability Statement should describe as completely as possible the process needed to reproduce the results of the study.
+
+A Data Availability Statement is usually required for a journal publication; however, the same concept is very useful at the completion of a project, or your PhD thesis, especially if you are leaving your institution. In such a case is a good idea to also indicate what data will be still available on the local server, how can be accessed who will be managing from now on and where more detailed information on this data, as a [data management plan](dmp.md), is available.
+
+This statement is different from a published data paper which usually includes a lot more information and refers to a specific dataset. A Data Availability Statement includes only how to access the data, but it refers to all the datasets used in the study.
+
+## Examples
+We copied here a few examples from the [American Meteorological Society website](https://www.ametsoc.org/index.cfm/ams/publications/author-information/formatting-and-manuscript-components/data-availability-statement-examples/), to show what is usually included. If you are doing this at the end of a project and you are leaving behind data on the local server, make sure someone will be managing it and put all relevant information in a data management plan, then list this too.
```

Comment:
I'm putting this in as :
".. leaving behind data on the local server, appoint a new data custodian, and put all relevant .."  otherwise is really long.